tests/meson.build

```diff
@@ -0,0 +1,19 @@
+test_sources = {
+    'test-hwinfo': [
+        '../src/eins-hwinfo.c',
+        '../src/eins-hwinfo.h',
+        '../src/eins-boottime-source.c',
+        '../src/eins-boottime-source.h',
```

Comment:
Maybe you should build these 2 files once in the src/ directory as an uninstalled `static_library()`, then link it into both the `eos-metrics-instrumentation` program and this test program?

Reply:
Thanks, I've made that change.